js/exporter/exceljs/export_data_grid.js

```diff
@@ -41,31 +58,43 @@ const helpers = {
         });
 
         return options;
-    },
+    }
 
     _isFrozenZone(dataProvider) {
         return dataProvider.getHeaderRowCount() > 0;
-    },
+    }
 
-    _isHeaderCell(dataProvider, rowIndex) {
-        return rowIndex < dataProvider.getHeaderRowCount();
-    },
+    _isHeaderCell(rowIndex) {
+        return rowIndex < this.dataProvider.getHeaderRowCount();
+    }
+
+    _isInfoCell() {
+        return false;
+    }
 
     _allowToMergeRange() {
         return true;
-    },
+    }
 
-    _getLoadPanelTargetElement(component) {
-        return component.getView('rowsView').element();
-    },
+    _getAllFieldHeaders() {
+        return [];
+    }
 
-    _getLoadPanelContainer(component) {
-        return component.getView('rowsView').element().parent();
+    _customizeCell(customizeCell, excelCell, gridCell) {
+        if(isFunction(customizeCell)) {
+            customizeCell(this._getCustomizeCellOptions(excelCell, gridCell));
+        }
     }
-};
+
+    _exportFieldHeaders() {}
+
+    _exportAllFieldHeaders() {}
+
+    _isRowFieldHeadersRow() {}
+}
 
 function exportDataGrid(options) {
-    return Export.export(_getFullOptions(options), helpers);
+    return Export.export(_getFullOptions(options), DataGridHelpers, _getLoadPanelTargetElement, _getLoadPanelContainer);
```

Comment:
Why did u decide to move `_getLoadPanelTargetElement` and `_getLoadPanelContainer` functions from the `Helpers`?

Reply:
I need initialize helpers after data are loaded. Functions `_getLoadPanelTargetElement` & `_getLoadPanelContainer` are using before.